tools/debug-ui/src/api.tsx

```diff
@@ -293,6 +293,64 @@ export interface RecentOutboundConnectionsResponse {
     };
 }
 
+export type Dropped = 'HeightProcessed' | 'TooManyProcessingBlocks';
+
+export type BlockProcessingStatus =
+    | 'Orphan'
+    | 'WaitingForChunks'
+    | 'InProcessing'
+    | 'Accepted'
+    | 'Error'
+    | Dropped
+    | 'Unknown';
+
```

Comment:
I think enums are encoded like this:

```suggestion
export type DroppedReason = 'HeightProcessed' | 'TooManyProcessingBlocks';

export type BlockProcessingStatus =
    | 'Orphan'
    | 'WaitingForChunks'
    | 'InProcessing'
    | 'Accepted'
    | {Error: string}
    | {Dropped: DroppedReason}
    | 'Unknown';

```

Reply:
Thanks, updated as suggested. 

to be honest, I don't really know how any of these encoding works, I just referred to the existing formats and followed them. However, the existing enum encoding in this file did not have an enum within an enum, so this was something I just did by trial and error.